include/osquery/sdk.h

```diff
@@ -87,7 +78,7 @@ REGISTER_INTERNAL(ExternalSQLPlugin, "sql", "sql");
   extern "C" EXPORT_FUNCTION void initModule(void);                            \
   struct osquery_InternalStructCreateModule {                                  \
     osquery_InternalStructCreateModule(void) {                                 \
-      Registry::declareModule(                                                 \
+      Registry::get().declareModule(                                           \
```

Comment:
NB: All `Registry` and `RegistryFactory` (they are aliases) methods should be non-static so we can decorate with const-ness. This will allow us to move from shared pointers to unique and weak later on, where *weak* will be used for tests.